packages/sdk/src/application/ApplicationClient.ts

```diff
@@ -438,31 +437,55 @@ export class ApplicationClient extends APIResource {
         await this._client.post<RestartPlaygroundConversationResponse>(
           queryString,
         );
-      return response;
+      return Promise.resolve(response);
     } catch (error) {
       return Promise.reject(error);
     }
   }
 
-  async chat(props: ChatRequest): Promise<ChatResponse> {
-    const { ownerId, appId, ...payload } = props;
+  async chat(
+    props: ChatRequest,
+    stream: boolean = true,
+  ): Promise<ReadableStream<Uint8Array>> {
+    const { ownerId, appId, requesterUid, accessToken, ...payload } = props;
+
+    if (
+      !accessToken ||
```

Comment:
@thewbuk Access token is completely not necessary, is is initize from the constructor of the ApiClient class. Ask user to input it again is a duplicated behavior